test/components/plots/barPlotTests.ts

```diff
@@ -64,18 +64,18 @@ describe("Plots", () => {
         var bar1 = d3.select(bars[0][1]);
         assert.closeTo(TestMethods.numAttr(bar0, "width"), xScale.rangeBand(), 1, "bar0 width is correct");
         assert.closeTo(TestMethods.numAttr(bar1, "width"), xScale.rangeBand(), 1, "bar1 width is correct");
-        assert.equal(bar0.attr("height"), "100", "bar0 height is correct");
-        assert.equal(bar1.attr("height"), "150", "bar1 height is correct");
+        assert.strictEqual(+bar0.attr("height"), 100, "bar0 height is correct");
+        assert.strictEqual(+bar1.attr("height"), 150, "bar1 height is correct");
         assert.closeTo(TestMethods.numAttr(bar0, "x"), 111, 1, "bar0 x is correct");
```

Comment:
Since we are comparing numbers, we should do everything in terms of numbers.

One argument for this from my view is that if we wanted to do any math, then we should definitely have numbers.  Otherwise we will have to do a bunch of coercion, going from string to number to calculate and then back to string